test/test_site.rb

```diff
@@ -508,9 +497,9 @@ def convert(*_args)
                                             source_dir, "_data", "categories.01", "dairy.yaml"
                                           ))
 
-        assert_equal site.data["categories01"]["dairy"], file_content
+        assert_equal site.data["categories.01"]["dairy"], file_content
```

Comment:
Why did you change the test here?

Reply:
This was one of the major things I had a concern about and was unsure how to address.

Due to the substitution changes/removal from data_reader we have this naming of property change to consider.

I think we might be able to address this though so the only breaking change is the index of the collections property on `site` instead.

Reply:
Looking it over its likely a fix would need to exist in the data_drop.rb but I was unsure what the change should be. If not there, then the sanitize_Filename method that got removed would be the right place. 

Either way, I think this is an avoidable breaking change given someone that is a bit more familiar with the sanitization we are doing. I do however like the idea that this makes data collections inline with the property references of  a normal collection, which means if you find yourself using data collection for too much and decide to break it out into its own collection later on then you have much less to worry about in terms of referencing the data.